### PR TITLE
added Ubuntu support

### PR DESCRIPTION
**Pull request by samvdb**
Added simple apt-get for ubuntu

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -1,7 +1,12 @@
 ---
+# Install logstash
```
Comment (ramondelafuente) [inline tasks/main.yml]: Ehhmm.. what exactly are we installing here? ;-)

File: tasks/main.yml [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
+- include: Debian.yml
+  when: ansible_distribution in ['Debian']
+
+# Install logstash
```
Comment (ramondelafuente) [inline tasks/main.yml]: Ditto...

**Comment by samvdb** [comment]
Oh damn, didnt check before copy paste :)

**Comment by samvdb** [comment]
Done. @ramondelafuente

**Comment by ramondelafuente** [comment]
Version bumped to v1.0.0 in galaxy

**Pull request merged**